app/helpers/peek_helper.rb

```diff
@@ -0,0 +1,13 @@
+module PeekHelper
+  def peek_available?
+    defined? Peek
```

Comment:
When would be Peek not be available ? Just asking because I'm still not sure about this 😕